Source/WebCore/PlatformHaiku.cmake

```diff
@@ -96,7 +96,7 @@ list(APPEND WebCore_SOURCES
   platform/graphics/texmap/GraphicsLayerTextureMapper.cpp
   platform/graphics/texmap/TextureMapperImageBuffer.cpp
 
-  platform/graphics/texmap/coordinated/CoordinatedGraphicsLayer.cpp
+  #platform/graphics/texmap/coordinated/CoordinatedGraphicsLayer.cpp
```

Comment:
What's the problem with this file?

Reply:
This file does not do anything everything is wrapped under USE(COORDINATEDGRAPHICS) while we have disabled it so thought it would be a good clean up 😅😅